README.md

```diff
@@ -138,8 +138,10 @@ The following configuration keys are supported:
     server is on a non-standard port. See: [wrap-ssl-redirect][9].
 
   - `:xss-protection` -
-    Enable the X-XSS-Protection header that tells supporting browsers
-    to use heuristics to detect XSS attacks. See: [wrap-xss-protection][10].
+    **Deprecated** Enable the X-XSS-Protection header. This is [no
+    longer considered best practice](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection)
```

Comment:
Can you use the "footnote" syntax for this link, to make it consistent with the rest of the README? So:

    This is [no longer considered best practice][13]

Reply:
Changed as suggested.